recipe/build.sh

```diff
@@ -2,6 +2,30 @@
 
 set -ex
 
+EXTRA_CMAKE_ARGS=()
+if [[ "$build_variant" == "qt" ]]; then
+  EXTRA_CMAKE_ARGS+=(
+    "-DWITH_QT:BOOL=ON"
+  )
+else
+  # Common between EGL and OSMesa
+  # We're linking the GLEW library that is included in VTK because it is compiled
+  # with EGL and OSMesa.
+  EXTRA_CMAKE_ARGS+=(
+    "-DGLEW_INCLUDE_DIR:PATH=${PREFIX}/include/vtk-9.1/vtkglew/include/GL/"
```

Comment:
Is there a way to parse the vtk version here? Otherwise we'll need to manually change the paths every time a new version of vtk is released.

Reply:
Great suggestion! I have just added a line to extract the version.